validator/Cargo.toml

```diff
@@ -23,4 +23,4 @@ solana-vote-api = { path = "../programs/vote_api", version = "0.16.0" }
 solana-vote-signer = { path = "../vote-signer", version = "0.16.0" }
 
 [features]
-cuda = []
+cuda = ["solana/cuda"]
```

Comment:
It was somehow failing to build while testnet deployment with `cuda = ["solana/cuda"]`.

Reply:
Do you remember why or how to repro?   Without this core/ doesn't get the cuda feature at all

Reply:
I was just running net.sh script that also builds the code.

Reply:
I just ran:
```
$ ./net.sh start -f cuda
```
with this PR.  Worked for me, and I see this in the validator log:
```
[2019-06-17T17:51:30.788625871Z WARN  solana::validator] CUDA is enabled
```

Reply:
It should be fine then. I was not even able to compile it. Maybe some other issue was causing my failures.